File: 01cpp/Lecture-4/pattern2.cpp
#include<iostream>
#include<stdlib.h>

using namespace std;

int main(){
    system("CLS");
    int r, c;
    cout<<"Enter the number of rows: ";
    cin>>r;
    cout<<"Enter the number of columns: ";
    cin>>c;
    int i=1, j=c;
    while(i<=r){
        while(j>0){
            cout<<j<<" ";
            j--;
        }
        j=c;
        i++;
        cout<<endl;
    }
}

File: 02python/lecture-4/pattern2.py
import os

def main():
    r,c=[int(x) for x in input("Enter the number of rows and columns: ").split()]
    i=1
    j=c
    while i<=r:
        while j>0:
            print(j,end=" ")
            j=j-1
        j=c
        i=i+1
        print()

if __name__=="__main__":
    os.system("cls")
    main()